Assets/Scripts/MiniGames/Song/ArrowSpawnerController.cs

```diff
@@ -39,14 +39,17 @@ IEnumerator ArrowSpawn()
         // arrow = Instantiate(arrowToSpawn, spawnLoc.position, spawnLoc.rotation) as Transform;
         // arrow.parent = spawnParent;
 
-        arrow = Instantiate(arrowToSpawn, spawnLoc.transform.position, spawnLoc.transform.rotation) as GameObject;
-        arrow.transform.SetParent(spawnParent.transform);
+      
 
-        callFunction = false;
+			callFunction = false;
             yield return new WaitForSeconds(Random.Range(minTimeBetweenSpawn, maxTimeBetweenSpawn));
 			callFunction = true;
-           // }
-        }
+		arrow = Instantiate(arrowToSpawn, spawnLoc.transform.position, spawnLoc.transform.rotation) as GameObject;
+		if (spawnParent != null) //get rid of null ref at the games end
+			{ arrow.transform.SetParent(spawnParent.transform); }
```

Comment:
we use braces like this for if blocks. should stay consistent with the rest of the code

````
if (spawnParent != null) {
    arrow.transform.SetParent(spawnParent.transform);
}
````